pages/pipelines/group_step.md

```diff
@@ -174,9 +174,9 @@ steps:
 
 ## Group merging
 
-If you upload a pipeline that has a `group` or `label` that matches a `group` or `label` in the pipeline that uploaded it, those groups will be merged together in the Buildkite UI.
+If you upload a pipeline that has a `group` or `label` that matches _the group of the step that uploaded it_, those groups will be merged together in the Buildkite UI.
```

Comment:
Thanks very much for this @mhaylock !

Since our [writing style guide](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#ui-elements) currently suggests using italics to refer to reference Buildkite UI elements in our docs, I'm trying to avoid extensive use of italics for other purposes (probably with the only exception being text that defines new terms).
The use of italics is a non-standard way for representing UI elements in documentation and at some point in the near future (when I have a little more bandwidth), I'd like to change this to bold, which is a much more common standard used by Google and Microsoft.

Therefore, I'd suggest removing italics here unless you feel it really adds value.

```suggestion
If you upload a pipeline that has a `group` or `label` that matches the group of the step that uploaded it, those groups will be merged together in the Buildkite UI.
```

Reply:
Thanks @gilesgas, that's very reasonable - I think it'll be fine without the emphasis so I've removed the italics as suggested 👍

Reply:
Thanks @mhaylock , I've already approved your update :-) Please feel free to merge this once the build is completed.